lang/csharp/Directory.Build.targets

```diff
@@ -0,0 +1,19 @@
+<!--
```

Comment:
I put them in with the reflect work so that 'dotnet test' would just run the test projects and not try to run tests in the other projects that didnt have them. I'll check whether they still help with this and let you know.

Reply:
Just tested. They are no longer doing anything and 'dotnet test' runs fine. Maybe it was something to do with ipctest. In any case, I removed them.